cmd/clusterctl/client/config/providers_client.go

```diff
@@ -315,6 +320,13 @@ func (p *providersClient) defaults() []Provider {
 			providerType: clusterctlv1.ControlPlaneProviderType,
 		},
 
+		// IPAM providers
```

Comment:
Let's also add the provider to the glossary etc.

For reference: https://github.com/kubernetes-sigs/cluster-api/pull/9012 (should be ~ the same)